Public/Src/Tools/BuildXL.Tools.CredentialProvider/README.md

```diff
@@ -0,0 +1,15 @@
+This project should be built in our pipeline to generate the CredentialProviderBuildXL.exe before each internal build. 
+The NUGET_CREDENTIALPROVIDERS_PATH should be updated to point to the latest CredentialProviderBuildXL.exe.
+
+Notes for future changes:
+
+Please use modify the url in the following command and test your newly generated exe file for your changes:
+CredentialProviderBuildXL.exe -uri https://pkgs.dev.azure.com/cloudbuild/_packaging/BuildXL.Selfhost/nuget/v3/index.json -verbosity detailed
+
+To build the BuildXL.Tools.CredentialProvider project locally for testing simply run "dotnet build -r win10-x64". 

```

Comment:
Let's use the portable `win-x64` runtime instead of `win10-x64`.  

[https://docs.microsoft.com/en-us/dotnet/core/rid-catalog#windows-rids](https://docs.microsoft.com/en-us/dotnet/core/rid-catalog#windows-rids)

Also update the build definition to use the win-x64 runtime